jax_distribution/logger.py

```diff
@@ -19,26 +19,24 @@
 from os.path import abspath, dirname
 from typing import Callable, Dict
 
+import chex
 import jax.numpy as jnp
 import numpy as np
 from colorama import Fore, Style
 from sacred.run import Run
 
-from jax_distribution.types import ExperimentOutput
 from jax_distribution.utils.logger_tools import Logger
 
 
-# todo: flake is complaining, possibly split into 3 functions
-#  log_absolute, log_trainer, log_evaluator
-def get_logger_fn(logger: Logger, config: Dict) -> Callable:  # noqa: CCR001
+def get_logger_fn(logger: SacredLogger, config: Dict) -> Callable:  # noqa: CCR001
```

Comment:
I think revert this file as the old stuff is correct, assume it is still being merged

Reply:
Seems like there was a mix up in the merge, definitely revert this file